base/applications/charmap/charmap.c

```diff
@@ -284,10 +286,14 @@ ChangeView(HWND hWnd)
     UINT xPos, yPos;
     UINT Width, Height;
     UINT DeskTopWidth, DeskTopHeight;
+    HWND hCopy;
```

Comment:
Only used when REMOVE_ADVANCED is defined.

Reply:
Sounds good, I'll prepare an update a bit later today to address this.  Thanks for the comments!